app/admin/admin.js

```diff
@@ -1,6 +1,8 @@
+"use client";
+
 import dynamic from "next/dynamic";
 
-const CMS = dynamic(() => import("../components/CMS"), { ssr: false });
+const CMS = dynamic(() => import("../../components/CMS"), { ssr: false });
```

Comment:
We could move components which are only used on single page into the respective `app` directory. This would make it bit clearer where things are used.